doc/cookbook/source/examples/classifier/multiclass_linearmachine.rst

```diff
@@ -0,0 +1,43 @@
+==========================
+Multi-class Linear Machine
+==========================
+
+We expand the application of linear machines to multi-class dataset by constructing generic multiclass classifiers with ensembles of binary classifiers.
+
+In this example, we show how to apply :sgclass:`CLibLinear` to multi-class cases with :sgclass:`CLinearMulticlassMachine`.
+
+See :cookbook_classifier:`linear_svm` for the infomration about :sgclass:`CLibLinear` binary classifier.
+
+-------
+Example
+-------
+
+Imagine we have files with training and test data. We create CDenseFeatures (here 64 bit floats aka RealFeatures) and :sgclass:`CMulticlassLabels` as
+
+.. sgexample:: multiclass_linearmachine.sg:create_features
+
+We use :sgclass:`CLibLinear` as base classifier. We create an instance of :sgclass:`CLibLinear` with L2 regularized classification. Then we set the residual convergence parameter and enable the bias.
```

Comment:
Just leave these parameters set to default values (this is already covered in the other cookbook), and don't talk about them here